R/spectrum_inputs.R

```diff
@@ -252,7 +255,8 @@ prepare_leapfrog_projp <- function(pjnz, hiv_steps_per_year = 10L, hTS = 3) {
   v$age15hivpop <- projp$age15hivpop
 
   ## Use Beer's coefficients to distribution IRRs by age/sex
-  v$incrr_age <- apply(projp$incrr_age, 2:3, beers::beers_sub_ordinary)[16:81, , ] ## !! Hard coded
+  Amat <- eppasm:::create_beers(17)
```

Comment:
I'd like to avoid introducing this dependency if we can. Any objections to adding this function?
```
beers_open_ended <- function(x) {
  c(beers::beers_sub_ordinary(x[-length(x)]), x[length(x)])
}
```
and using that?

Reply:
Yep, that sounds good to me. Done in 4dd1068.